src/app/index.tsx

```diff
@@ -36,6 +37,7 @@ function App() {
         <CustomRoute component={Login} authenticated={logged} path={PATHS.login} exact />
         <CustomRoute component={SignUp} authenticated={logged} path={PATHS.signup} exact />
         <CustomRoute component={Home} authenticated={logged} path={PATHS.home} protect exact />
+        <CustomRoute component={BookDetail} authenticated={logged} path={PATHS.bookDetail} protect exact />
```

Comment:
remember about specificity of routes, this works because `home` is exact, but if it wasnt exact this route will never show.
Move this route upper home and remove exact, detail routes are not exact